superset/assets/src/SqlLab/components/SouthPane.jsx

```diff
@@ -39,6 +42,12 @@ class SouthPane extends React.PureComponent {
       latestQuery = props.editorQueries[props.editorQueries.length - 1];
     }
     let results;
+    if (this.props.offline){
+      return (
+        <Label className="m-r-3" bsStyle={STATE_BSSTYLE_MAP['offline']}>
+          offline
```

Comment:
instead of create a hard-coded text here, you should create new entry in STATUS_OPTIONS in constants.js